zowex/Cargo.toml

```diff
@@ -9,8 +9,9 @@ repository = "https://github.com/zowe/zowe-cli"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+atty = "0.2.14"
 pathsearch = "0.2.0"
 rpassword = "5.0.1"
 sysinfo = "0.21.1"
-serde_json  = "1.0.69"
-serde  = { version = "1.0.130", features = ["derive"]}
+serde_json = "1.0.69"
+serde = { version = "1.0.130", features = ["derive"]}
```

Comment:
Before Version 0.5.0 gets included in the Zowe Release, the BoM may need updated.

Reply:
When the next-daemon-enable branch is merged we will be at 0.6.0.